.github/workflows/release-stable-dry-run.yml

```diff
@@ -6,6 +6,18 @@ on:
     - cron: '0 2 * * 1-5'
 
 jobs:
+  dry-run-release-82:
+    name: "Release from stable/8.2"
+    uses: camunda/zeebe/.github/workflows/release.yml@stable/8.2
+    secrets: inherit
+    strategy:
+      fail-fast: false
+    with:
+      releaseBranch: stable/8.2
+      releaseVersion: 0.8.1
```

Comment:
❌ You need a distinct version for each job, otherwise the artifact upload might encounter conflicts (I remember I had those during testing)
```suggestion
      releaseVersion: 0.8.2
```